pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -29,6 +29,8 @@
 class EvaluationLoop(DataLoaderLoop):
     """Loops over all dataloaders for evaluation."""
 
+    name = "val_loop"
```

Comment:
why these names?

this is not the val loop, this is the evaluation loop for both testing and validation.

Reply:
Removed !